db/seeds.rb

```diff
@@ -70,7 +82,8 @@
       1 => ['hashiru', 'run', 'ran', 'run'],
       2 => ['iu', 'say', 'said', 'said']
     }.to_json,
-    work_group: work_groups[number]
+    work_group: work_groups[number],
+    name: "Past Tense #{{number + 1}}"
```

Comment:
I think this will need to be change too
the original idea is that each group has its own worksheet, but then it's true that we would end up with worksheets with the same name
maybe it makes more sense to change to an N-N relationship

Reply:
@Naokimi updated the seeds! Please have a look.